core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -105,6 +105,14 @@
       + "<code>MetricReporter</code> interface allows plugging in classes that will be notified "
       + "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+
+  public static final String METRICS_TAGS_CONFIG = "metrics.tags.map";
```

Comment:
double plural sounds weird, but may be right.
"metric.tags.map" or "metrics.tag.map" reads better